OracleSOASuite/dockerfiles/12.2.1.4/README.md

```diff
@@ -195,20 +334,72 @@ Sample data for `osbserver.env.list` will look like this:
     MANAGED_SERVER_CONTAINER=true
     MANAGEDSERVER_PORT=9001
 
-A sample docker run command is given below:
+Example for `osbserver2.env.list`:
+
+    MANAGED_SERVER=osb_server2
+    DOMAIN_TYPE=osb
+    DOMAIN_NAME=soainfra
+    ADMIN_HOST=<Administration Server hostname>
+    ADMIN_PORT=7001
+    ADMIN_PASSWORD=Welcome1
+    MANAGED_SERVER_CONTAINER=true
+    MANAGEDSERVER_PORT=9002
+
+
+To start a Docker container for the Oracle Service Bus server (for `osb` domain type), you can use the `docker run` command passing `osbserver1.env.list`.
+
+For example:
+
+    $ docker run -i -t  --name soams1 --network=SOANet -p 9001:9001  -v $data_volume/SOA:/u01/oracle/user_projects  --env-file ./osbserver1.env.list oracle/soasuite:12.2.1.4.0 "/u01/oracle/container-scripts/startMS.sh"
+
+Similarly, to start a second Docker container for the Oracle Service Bus server (for `osb_server2` of `osb` domain type), you can use the `docker run` command passing `osbserver2.env.list`.
+
+For example:
+
+    $ docker run -i -t  --name soams2 --network=SOANet -p 9002:9002  -v $data_volume/SOA:/u01/oracle/user_projects  --env-file ./osbserver2.env.list oracle/soa:12.2.1.4.0 "/u01/oracle/container-scripts/startMS.sh"
+
+The following lines indicate when the Oracle Service Bus Managed Server is ready to be used:    
+    INFO: Managed Server is running
 
-    $ docker run -i -t  --name soams --network=SOANet -p 9001:9001  --volumes-from soaas  --env-file ./osbserver.env.list oracle/soasuite:12.2.1.4 "/u01/oracle/container-scripts/startMS.sh"
+    INFO: Managed Server has been started
 
->IMPORTANT: Note that the container port used for OSB server (9001) is different from SOA server (8001).
+Once the Managed Server container is created, you can view the server logs:
 
-Now you can access
+    $ docker logs -f \<Managed Server container name\>
 
-   * AdminServer Web Console at http://\<hostname\>:7001/console  with weblogic/Welcome1 credentials.
+
+### 8. Access the Consoles
+Now you can access the following Consoles:
+
+   * Administration Server Web Console at http://\<hostname\>:7001/console  with weblogic/Welcome1 credentials.
```

Comment:
Please remove all the blank lines between your list items and remove the indentation.